src/index.js

```diff
@@ -4,9 +4,20 @@ const path = require('path');
 const util = require('util');
 const TapMochaReporter = require('tap-mocha-reporter');
 
-async function findBrsFiles(sourceDir) {
-    let searchDir = sourceDir || 'source';
-    const pattern = path.join(searchDir, '**', '*.brs');
+async function findBrsFiles(exclusions) {
+    const defaultDirs = ['source', 'components'];
+    let exclusionList = [];
+
+    if (exclusions) {
+        exclusionList = exclusions.trim().split(',');
+    }
+
+    let inclusionList = defaultDirs
+        .filter( dir => !exclusionList.includes(dir))
+        .map( dir => path.join(dir, '**', '*.brs'));
+
+    let pattern = inclusionList.length > 1 ? `{${inclusionList.join(',')}}` : inclusionList[0];
```

Comment:
Might want to consider reducing this to `inclusionList.join(',')`.

Reply:
That won't work when there is only one inclusion in the array because it will be surrounded by braces `{}` which for some reason only work if there are multiple elements.
I also don't think this is most elegant way to do this but haven't got a better idea